docs/using/backup.md

```diff
@@ -1,26 +1,24 @@
 # Backup and Restore
 
 !!! caution alert alert-warning "Caution"
-    Backup and restore features are a technical preview and currently only work with MySQL database servers backing up to Amazon AWS S3 storage locations.  You must also enable this feature at container creation time by adding `-e ENABLE_BACKUP_MANAGEMENT=1` to your `docker run` command. After that, go to `/graph/backup` to check the Backup Management page.
+    Backup and restore features are a technical preview and currently only works with a MySQL database server or a MongoDB replica set cluster, backing up to an Amazon AWS S3 storage location.
+
+!!! summary alert alert-info "Summary"
+    - Enable backup features.
+    - Add a storage location.
+    - Satisfy preconditions:
+        - For [MySQL](#mysql-backup-preconditions):
+            - Confirm instance service parameters and storage location.
+            - Install required packages.
+        - For [MongoDB](#mongodb-backup-preconditions):
+            - Install [Percona Backup for MongoDB] on every node in the replica set.
```

Comment:
PMM expects that pbm-agent started on each node. So better to make it clear that pbm-agent should be started by the user. 
```suggestion
            - Install and run [Percona Backup for MongoDB] on every node in the replica set.
```